packages/next/build/webpack-config.ts

```diff
@@ -888,7 +888,10 @@ export default async function getBaseWebpackConfig(
               // pre-webpack era (common in server-side code)
               'global.GENTLY': JSON.stringify(false),
             }
-          : undefined),
+          : {
+              // Replace `globalThis` with `window` in browser environments
+              globalThis: 'window',
```

Comment:
This will very likely break a lot of dependencies that run in workers for example.

Reply:
Thank you for the quick review! I may remove it until a better solution arises.